README.md

```diff
@@ -296,6 +301,54 @@ The BApp (Blockchain Application) Development sample projects using caver-js are
 * [Count BApp](https://docs.klaytn.com/bapp/tutorials/count-bapp)
 * [Klaystagram](https://docs.klaytn.com/bapp/tutorials/klaystagram)
 
+## Trouble shooting and known issues
+
+### Connect with Kaikas Web Extension
+
+Kaikas Web Extension Wallet works well only up to [caver-js v1.4.1](https://www.npmjs.com/package/caver-js/v/1.4.1).
+Features provided in later versions (caver-js v1.5.0~) may not work with Kaikas Web Extension Wallet.
+If the error below occurs when connecting to Kaikas, use caver-js v1.4.1.
+
+```
+Kaikas only processes one transaction at a time. Open Kaikas and refresh the pending transaction. If the service doesn’t process your transaction for a while, cancel the pending transaction.
+```
+
+```
+Kaikas는 한 번에 하나의 트랜잭션만 처리합니다. Kaikas를 열어 대기 중인 트랜잭션을 새로고침 해주세요. 만약 대기 상태가 계속된다면 이용 중인 서비스가 트랜잭션을 처리하지 않는 것이니 트랜잭션을 취소바랍니다.
```

Comment:
Please remove this line. :)

Reply:
I intentionally added this(our repo should not have korean though) because we don't have separate korean version readme, but most of korean developers will use korean setting with Kaikas and message they will get will be this.